tests/conftest.py

```diff
@@ -115,18 +120,37 @@ def clean_serveradmin(filters):
     Query(filters).delete().commit()
 
 
+def clean_aws(vm_hostname):
+    try:
+        vm_stop(vm_hostname)
```

Comment:
the purpose of such tests is to test functions such as `vm_stop` and `vm_delete` and not to use them when not actually testing them. we can never rely on them because they can be broken anytime. the cleanup must be done "manually" without using any existing code. otherwise we might run into a hen-egg problem. please add some code here to hard-kill those vms on aws

Reply:
I reworked the cleanup for AWS, which should be fine now.